server/gitrest/lerna-package-lock.json

```diff
@@ -380,30 +380,38 @@
 			}
 		},
 		"@fluidframework/gitresources": {
-			"version": "0.1036.4000-68094",
-			"resolved": "https://registry.npmjs.org/@fluidframework/gitresources/-/gitresources-0.1036.4000-68094.tgz",
-			"integrity": "sha512-IrGYbtiV8M+nZCJ1n4Xs0VmhQgngMtwlE+vPMPUJhry2AcEUrEVBK5h9+EAfz2IIoVx+C+S5VVa6HShBmziA8Q=="
+			"version": "0.1037.1000-75673",
```

Comment:
@hedasilv can you or someone on your side confirm no issues with these (unrelated to the rest of this PR) lock changes?

Honestly I'm confused how it is not failing as-is.  The old version here doesn't match the pattern in gitrest's package.json file.